### DISCUSSION TITLE
Can a drawable draw on top of its content?

### DISCUSSION THREAD

**Discussion by WileyNg** [picoe/Eto#2298] [General]
Right now the OnPaint event always draws underneath its Content.
Can I draw, let's say a RoundedRectangle border around and on top of a dynamic control?

**Comment by WileyNg** [comment]
@cwensley  any idea about this?
Or should I use PixelLayout?

**Comment by cwensley** [comment]
Hey @WileyNg, yeah `PixelLayout` is currently the only way to make controls go overtop each other in `Eto`, unless you dive into some custom platform-specific code.
